app/code/Magento/CatalogImportExport/Model/Import/Product/Validator/Media.php

```diff
@@ -100,4 +98,12 @@ public function isValid($value)
         }
         return $valid;
     }
+    
+    /**
+     * @return string
+     */
+    protected function getMultipleValueSeparator()
```

Comment:
New methods and properties should be declared as `private` without strong reason to make them `protected` or `public`